Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -160,11 +166,26 @@ public struct CustomerCenterConfigData {
                 case .pleaseContactSupportToManage:
                     return "Please contact support to manage your subscription."
                 case .appleSubscriptionManage:
-                    return "You can manage your subscription by using the App Store app on an Apple device."
+                    return "You have an active Apple App Store purchase. " +
```

Comment:
```suggestion
                    return "You have an active subscription from the Apple App Store. " +
```

Reply:
Not sure if there's a case for a "purchase" being active that isn't a subscription that I'm not thinking of. "Active purchase" struck me funny because I don't know what "active" means in the context of a purchase. If "purchase" is necessary, I'd consider something more descriptive, like:

> You have a purchase from the Apple App Store that's still granting you access to this service.

Reply:
I was using purchase to cover consumables/lifetimes, but I was wrong becasue this view won't open if there are no subscriptions, so this should be subscription